### PR TITLE
Add codeowners for Workspaces-owned files (WOR-584).

### PR DESCRIPTION
**Pull request by cahrens**
I followed syntax as defined in https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax, but don't know how to actually test that it works before the PR is merged. Iterating if necessary doesn't seem like a big deal though.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -2,3 +2,12 @@
 # See https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax
 /src/libs/brand*.js @DataBiosphere/terra-cobranding
 /src/libs/colors.js @DataBiosphere/terra-cobranding
+/src/pages/billing/ @DataBiosphere/broadworkspaces
```
Comment (cahrens) [inline .github/CODEOWNERS:5]: Everything under billing (recursive)

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -2,3 +2,12 @@
...
+/src/pages/workspaces/* @DataBiosphere/broadworkspaces
```
Comment (cahrens) [inline .github/CODEOWNERS:6]: Files in workspaces, but not recursive (so just `List.js` at this point)

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -2,3 +2,12 @@
...
+/src/pages/workspaces/workspace/*Dashboard* @DataBiosphere/broadworkspaces
```
Comment (cahrens) [inline .github/CODEOWNERS:7]: Everything with Dashboard in the name (note that it is case sensitive).

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -2,3 +2,12 @@
...
+/src/pages/workspaces/workspace/*Workspace* @DataBiosphere/broadworkspaces
+/src/components/*Workspace* @DataBiosphere/broadworkspaces
+/src/components/workspace-utils* @DataBiosphere/broadworkspaces
```
Comment (cahrens) [inline .github/CODEOWNERS:10]: Leaving end as a wildcard in hopes that unit tests will exist later.

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -2,3 +2,12 @@
 # See https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax
 /src/libs/brand*.js @DataBiosphere/terra-cobranding
 /src/libs/colors.js @DataBiosphere/terra-cobranding
+/src/pages/billing/ @DataBiosphere/broadworkspaces
+/src/pages/workspaces/* @DataBiosphere/broadworkspaces
+/src/pages/workspaces/workspace/*Dashboard* @DataBiosphere/broadworkspaces
+/src/pages/workspaces/workspace/*Workspace* @DataBiosphere/broadworkspaces
```
Comment (marctalbott) [inline .github/CODEOWNERS:8]: Do we also own `RequestAccessModal.js` or is that Identiteam?

**Review by marctalbott — APPROVED** [review]

File: .github/CODEOWNERS (hunk 5 above)
Comment (cahrens) [reply .github/CODEOWNERS:8]: I thought IdentiTeam.

**Review by ghost — APPROVED** [review]

**Pull request merged**